corehq/pillows/utils.py

```diff
@@ -89,3 +93,17 @@ def get_all_expected_es_indices():
 def format_form_meta_for_es(form_metadata):
     form_metadata['appVersion'] = form_metadata['appVersion'].get('#text')
     return form_metadata
+
+
+def update_mapping_migration_op(index_info, quiet=True):
```

Comment:
What do you think of changing the name of this argument to `index` and passing in `index_info.index`?

Reply:
Good catch, previous (squashed) versions of this needed the info object. I missed this simplification after cleaning it up.